new/PRS12971GroundHornbill.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12971GroundHornbill" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gound Hornbill Artist</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JG"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JG" when="2020-04-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Gound Hornbill Artist</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <faith type="EOTC"/>
+                    <occupation type="art"/>
+                    <nationality type="Ethiopia"/>
+                    <birth notBefore="1650"/>
+                    <death notAfter="1750"/>
+                    <floruit ></floruit>
+                </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS12971GroundHornbill" passive="BDLaethe34"/>
```

Comment:
Could it be useful to add some more items painted by him?
As to "saws:isCopierOf", I think there should be a proper relation for an artist. I do not know whether it has been already identified @PietroLiuzzo  @thea-m

Reply:
I don't think we have one yet. We (@Gnisci ) should decide whether saws:isCopierOf is appropriate, and otherwise, create one (or use one existing in an ontology - I browsed https://ancientwisdoms.ac.uk/media/ontology/doc/index.html and saw nothing, but it might exist elsewhere? @PietroLiuzzo where would be a good place to look?)